eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -160,13 +160,13 @@ jobs:
         displayName: "Install dependencies"
 
       - script: |
-          node common/scripts/install-run-rush.js build --verbose $(GeneratedPackageTargets)
+          node common/scripts/install-run-rush.js build --verbose $(GeneratedPackageTargetsTo)
```

Comment:
Guessing this should be `GeneratedPackageTargetsFrom` ?

Reply:
I think the build step with `from` may still fail since it doesn't build transitive dependencies

Reply:
That's possible, but it's a bug in Rush. If we want to run `unit-test` on a set of targets, we need to run `build` and `build:test` on the same set of targets first. It doesn't help us to run `build` on a different set of targets than the other two commands use.

Reply:
Tested the ci pipelines for storage, keyvault, core, etc For the keyvault-ci, the build fails with only the `from` tag and succeeds with `to` tag. For other two, the build succeeds with `from` tag. I guess I can add both `from` and `to` tag to the build then?

Reply:
I think doing both for this build step here makes sense